src/Http/Controllers/SitemapController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Rapidez\Sitemap\Http\Controllers;
+
+use Illuminate\Http\Response;
+use Rapidez\Sitemap\Models\Sitemap;
+use TorMorten\Eventy\Facades\Eventy;
+
+class SitemapController
+{
+    public function index(): Response
```

Comment:
You *could* just make it invokable that way you don't have to specify the method when registering the route.